docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,104 @@
+
+In other words, when you call [`GetReadings()`](/components/movement-sensor/#getreadings) on a `merged` sensor, the `Readings` each sensors has taken are combined in one response from your robot.
+
+This is especially useful if you want to get readings of position and orientation *or* linear and angular velocity at the same time, which are normally separately supported and returned by [`GPS`](/components/movement-sensor/gps/) or [`IMU`]((/components/movement-sensor/gps/)) models, respectively.
+
+{{% alert title="Usage Tip: Navigation Service" color="tip" %}}
+Aggregating the readings of a [`GPS`](/components/movement-sensor/gps) and [`IMU`]((/components/movement-sensor/imu)) movement sensor in a `merged` model allows you to reduce velocity error when your robot is trying to [navigate](/services/navigation/) with these position and orientation readings.
+{{% /alert %}}
+
+Before configuring a `merged` movement sensor, configure each movement sensor you want to merge as an individual component according to its [model's configuration instructions](/components/movement-sensor/#configuration).
+Reference the `name` you configure for each individual component in the `merged` sensor's configuration attributes as shown:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your movement sensor, select the `movement-sensor` type, and select the `merged` model.
+
+Click **Create Component**.
+
+{{< imgproc src="/components/movement-sensor/merged-config-builder.png" alt="Creation of an `merged` movement sensor in the Viam app config builder." resize="600x" >}}
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-sensor-name>",
+      "type": "movement_sensor",
+      "model": "merged",
+      "attributes": {
+        "position": ["<your-gps-sensor-name-1>", "<your-gps-sensor-name-2>"],
+        "orientation" : ["<your-imu-sensor-name-1>"],
+        "compass_heading" : ["<your-gps-sensor-name-1>"],
+        "angular_velocity":["<your-angular-velocity-sensor-name-2>"],
+        "linear_velocity": ["<your-gps-sensor-name-1>"],
+        "linear_acceleration": ["<your-adxl345-sensor-name-1>"]
+      },
+    "depends_on": [] 
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// { "components": [ { ... },
+{
+    "name": "<your-sensor-name>",
+    "type": "movement_sensor",
+    "model": "merged",
+    "attributes": {
+        "position": ["gps1"],
+        "orientation" : ["vectornav"],
+        "compass_heading" : ["gps1"],
+        "angular_velocity":[""],
```

Comment:
Added vectornav, lmk if you want both in array to demonstrate

Reply:
good idea, and you can use that to demonstrate which one we'll pick (the first one since both report angular velocity and do not error since the driver and config is healthy)

Reply:
Oh smart, I'll add this to the end note as well

Reply:
added!